<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPass</class>
 <widget class="QWidget" name="ResetPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码重置</string>
  </property>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>重置密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>275</width>
     <height>118</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="id">
      <property name="text">
       <string>ID：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="id_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pass0">
      <property name="text">
       <string>新密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QLineEdit" name="pass0_edit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>输入新的密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QLabel" name="pass1">
      <property name="text">
       <string>再输一次：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pass1_edit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>再输入一次</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="pass_diff">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>144</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>两次密码不一致！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
